packages/components/src/badge/tests/chromatic/BadgeIcon.chroma.jsx

```diff
@@ -0,0 +1,52 @@
+import { PlaceholderMajorIcon } from "@components/icons";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong